### ISSUE TITLE
[bug] rename command in download_references

### ISSUE THREAD

**Issue by simone-pignotti** [bactopia/bactopia#184]
**Description**
The rename command at this line does not work with the standard `rename` command from `util-linux`; it needs the perl-based rename script which is not installed by default in many distros.
https://github.com/bactopia/bactopia/blob/c22d47099c1ca4f7c1565b3e0c06861d86101035/templates/download_references.sh#L38

**Steps to Reproduce**
In my case, this was discovered while running bactopia on AWS. It is weird because it had worked before and I can't figure out what changed, but my guess is that this would fail on any system without perl-based rename.

**Expected Behavior**
Should work without further dependencies.

**Execution Environment**

- Bactopia Version: 1.5.5
- OS: Amazon Linux 2
- Environment: conda

**Additional Information**
I have removed this line and replaced the previous one with:
```
find refseq -name "*.gbff.gz" | xargs -I % sh -c 'mv % genbank_temp/`basename % | sed -E "s/(GC[AF]_[0-9][0-9]*).*/\1/g"`'
```
The command works on both Linux and macOS and is supposed to be POSIX-compliant. I am testing it within bactopia now.

**Comment by simone-pignotti** [comment]
Thank you @rpetit3 ! No worries I can fix that manually, no need to push a new version only for this.

**Comment by simone-pignotti** [comment]
PS the alternative xargs-based command I suggested worked, in case you ever want to get rid of the perl dependency in `download_references`

**Comment by rpetit3** [comment]
Released v1.5.6, should be good to go.

Feel free to reopen if not!
